nodes/nodes-about-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,78 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="nodes-about-autoscaling-nodes"]
+= About autoscaling nodes on a cluster
+:context: nodes-about-autoscaling-nodes
+toc::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+Autoscaling is only available on clusters that were purchased through the Red Hat Marketplace.
+====
+endif::[]
+
+The autoscaler option can be configured to automatically scale the number of machines in a cluster.
+
+The cluster autoscaler increases the size of the cluster when there are pods that failed to schedule on any of the current nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
+
+Additionally, the cluster autoscaler decreases the size of the cluster when some nodes are consistently not needed for a significant period, such as when it has low resource use and all of its important pods can fit on other nodes.
+
+When you enable autoscaling, you must also set a minimum and maximum number of worker nodes.
+
+[NOTE]
+====
+Only cluster owners and organization admins can scale or delete a cluster.
+====
+
+[id="nodes-enabling-autoscaling-nodes"]
+== Enabling autoscaling nodes on a cluster
+
+Autoscale worker nodes on a cluster to increase or decrease the number of resources available. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
+
+[discrete]
+include::modules/ocm-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+
+ifdef::openshift-rosa[]
+[NOTE]
+====
+Additionally, you can configure autoscaling on the default machine pool when you xref:../rosa_getting_started/rosa-creating-cluster.adoc#rosa-creating-cluster[create the cluster using interactive mode].
+====
+
+[discrete]
+include::modules/rosa-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+endif::[]
+
+[id="nodes-disabling-autoscaling-nodes"]
+== Disabling autoscaling nodes on a cluster
+
+Autoscale worker nodes on a cluster to increase or decrease the number of resources available. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
+
+ifdef::openshift-dedicated[]
+You can disable autoscaling on a cluster using OpenShift Cluster Manager (OCM) console.
+endif::[]
+
+ifdef::openshift-rosa[]
+You can disable autoscaling on a cluster using OpenShift Cluster Manager (OCM) console or the {product-title} CLI.
+
+[NOTE]
+====
+Additionally, you can configure autoscaling on the default machine pool when you xref:../rosa_getting_started/rosa-creating-cluster.adoc#rosa-creating-cluster[create the cluster using interactive mode].
```

Comment:
Do you need this note here? You wouldn't need to disable autoscaling on a new cluster, right? And, you aren't interested in enabling autoscaling when looking for info on disabling. (Unless you want to disable it for the individual nodes, then enable it for the cluster.)